### ISSUE TITLE
Acoustics (SRAO:0000217)

### ISSUE THREAD

**Issue by only1chunts** [FAIRsharing/subject-ontology#32]
Currently its only within the Mechanical Engineering clade, but Acoustics refers to both sound and hearing, so should also be in the biology clade somewhere (not sure where though).

**Comment by allysonlister** [comment]
Thanks very much for this suggestion. You're correct that Acoustics should have more parents. AgroVoc has Acoustics as a child of Physics (see http://agrovoc.uniroma2.it/agrovoc/agrovoc/en/page/c_49911) as do other resources such as OMIT and PLoS Thesaurus. We will refactor Acoustics to align with AgroVoc during this point release as well as adding Physics as a parent term. Any additional hierarchy placements will be considered this week and this ticket will be updated accordingly. 

Thanks again!

**Comment by allysonlister** [comment]
Pete and I discussed Acoustics yesterday and, as a result, we have added the parent term 'Physics'. We have investigated the placement of this term in a number of CVs and looked into various definitions (e.g. https://en.wikipedia.org/wiki/Acoustics). We are happy that this subject area concerns the physical shape and structure of an environment / room, and does not directly dealing with biology. 

If there is a specific field of biology / sound / hearing that you wish to be included in SRAO, we would love for you to create an additional ticket with that suggestion. 

Thanks again for your comments! The changes from this ticket are visible now in the development version of SRAO, and will be published as part of the 0.2.0 point release.
